src/main/java/com/example/template/models/Patient.java

```diff
@@ -0,0 +1,18 @@
+package com.example.template.models;
+
+import java.sql.Date;
+
+import org.springframework.data.annotation.Id;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class Patient {
+  @Id private String id;
```

Comment:
I think mongo `ObjectId` is better than string.

Reply:
Pls ignore my comment. I think @gregory-akins changed it to String for some reason in measure model(measure service). You can leave it as string here as well unless someone has any objection.

Reply:
@adongare Ok thanks so much for letting me know! Appreciate it!!